### ISSUE TITLE
Console Error after time end

### ISSUE THREAD

**Issue by SNokerYT** [divios/DailyShopIssues#4]
Console Error after time end
[13:52:22 WARN]: net.minecraft.server.CancelledPacketHandleException
![advancedenchants3f132d](https://user-images.githubusercontent.com/68678154/131216971-a1de49a3-c5b8-4065-a6fa-2a8925c952b5.gif)

**Comment by divios** [comment]
You are absolutely right, i didn't test this myself and i was calling something asynchronously that messed up things. I also spotted another bug doing the debug so great, thanks!
